rule_packages/c/Contracts1.json

```diff
@@ -0,0 +1,48 @@
+{
+  "CERT-C": {
+    "ENV30-C": {
+      "implementation_scope": {
```

Comment:
if there are no implementation notes, do we need this section in the json?

Reply:
oh, so is this actually an issue on the rule description generation, like a good default would be to include that section in each json with a none value? although we can open a separate issue for this and address later